products/sle/main.pm

```diff
@@ -769,7 +769,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
         }
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
-        loadtest "virt_autotest/reboot_and_wait_up_normal";
+        loadtest "virt_autotest/reboot_and_wait_up_normal" unless is_sle('>=15-sp2');
```

Comment:
I am afraid reboot is only not necessary for developing product.

Reply:
Yes, I think so. change to apply to developing only and verify

only verify the scheduled modules:
[gi-guest_developing-on-host_developing-xen](http://10.67.129.51/tests/1627)
[gi-guest_developing-on-host_sles15sp2-xen](http://10.67.129.51/tests/1628)
[gi-guest_developing-on-host_sles12sp5-kvm](http://10.67.129.51/tests/1629)